srml/eth-backing/src/tests.rs

```diff
@@ -55,3 +57,113 @@ fn verify_parse_token_redeem_proof() {
+					.from_hex().unwrap(),
+				header_hash: H256::from(hex!("f3cc3fab1b6cae48660a36839630c350bace54156d57ee3c62c6113d4b7d82b1"))
+			};
+
+			let mixh = H256::from(hex!("7afe3c56ba983149cc5690df75110c5b8bd108d99ffb3203ea94d1bb0811389f"));
+			let nonce = H64::from(hex!("7775c8bc9f155252"));
+
+			let header = EthHeader {
+				parent_hash: H256::from(hex!("e81c2b775e2fe499fc108626ac8fdb427eca0ef4073c4737ab85e4ad77245d2f")),
+				timestamp: 0x5df8dc97,
+				number: 6983947,
+				author: EthAddress::from(hex!("d34912efb0e7fedaedb9390990d7ef623e01f4fa")),
+				transactions_root: H256::from(hex!("2c1a476b3bb42bccd51f3df35c25cb1167de017f13c086b9d58dc56f2366614f")),
+				uncles_hash: H256::from(hex!("1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347")),
+				extra_data: "706f6f6c696e2e636f6d".from_hex().unwrap(),
+				state_root: H256::from(hex!("aa6b1f9de1b3acf0939928b09aed7177bb81ea3bc102d05ce6fdada0fb8ca11c")),
+				receipts_root: H256::from(hex!("162102c848b94ffb7e3768f9df5df461da28f63d8f9240484246c037bb8f7460")),
+				log_bloom: Bloom::from_str("242800210084001820600020041104044004018450430c10081080a01224000920e434b1082288020100080042400028802504000208884d041009203036121019c0004117072068c1020088006a004401420421091400088120375400642008218030c8228102041410100000308a9c090804292800880049008111010280250900180869208025160238400000a04040630730c0d184004042440120ac0000220a000000c811810202010440040211d020c60140a8021a040824040110416000a0682300800010001980000094081846d221130428800803a3830c4420603206d0000c040014920402080008009020840f6e4c608d41420420080000000142").unwrap(),
+				gas_used: 0x78bd9a.into(),
+				gas_limit: 0x79d4fe.into(),
+				difficulty: 0x75e5a3ef_u64.into(),
+				seal: vec![rlp::encode(&mixh), rlp::encode(&nonce)],
+				hash: Some(H256::from(hex!("f3cc3fab1b6cae48660a36839630c350bace54156d57ee3c62c6113d4b7d82b1"))),
+			};
+
+			assert_ok!(EthRelay::init_genesis_header(&header, 0x68de130d2c02a8_u64));
+
+			let expect_account_id = <Test as Trait>::DetermineAccountId::account_id_for(&hex!("2a92ae5b41feba5ee68a61449c557efa9e3b894a6461c058ec2de45429adb44546")).ok().unwrap();
```

Comment:
`ok()` does nothing here.

Reply:
Fixed in https://github.com/darwinia-network/darwinia/pull/172/commits/07694a353093cad2723c3f08001d702c4fe8664b